src/lxc/cgroups/cgfsng.c

```diff
@@ -232,10 +232,11 @@ static void append_line(char **dest, size_t oldlen, char *new, size_t newlen)
 /* Slurp in a whole file */
 static char *read_file(const char *fnam)
 {
-	FILE *f;
-	char *line = NULL, *buf = NULL;
-	size_t len = 0, fulllen = 0;
+	__do_fclose FILE *f;
```

Comment:
I see that you don't need it here right now, but in general I think adding = NULL; here would be good, in case someone ends up adding an if (solstice) return NULL;  before the f = fopen().

Reply:
Agreed.